vcd/datasource_vcd_nsxt_alb_settings.go

```diff
@@ -42,6 +42,12 @@ func datasourceVcdAlbSettings() *schema.Resource {
 				Computed:    true,
 				Description: "Optional custom network CIDR definition for ALB Service Engine placement (VCD default is 192.168.255.1/25)",
 			},
+			"supported_feature_set": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
No need to set for `Optional: true` in the datasource. We never accept this as a parameter.

Reply:
Yeah, you're right. Removed.